diff.py

```diff
@@ -2163,8 +2178,10 @@ def lo_hi_match(old: str, new: str) -> bool:
         new_inner = new[new_idx + 4 : -1]
         return old_inner.startswith(".") or new_inner.startswith(".")
 
-    def diff_sameline(old: str, new: str) -> None:
+    def diff_sameline(old_line_obj: Line, new_line_obj: Line) -> None:
```

Comment:
no need for an _obj suffix

Reply:
the reason I have it is because this function now contains old as a string, new as a string, and then two additional old new vars that are the actual object...   is it okay to just have those called old_line and new_line?

then we will have old (str), new (str), old_line (Line) new_line (Line)

I'll put it to whatever you decide.

Reply:
`old_line`/`new_line` are good names imo; they match the `Line` type